x-pack/plugins/security_solution/public/flyout/document_details/left/index.tsx

```diff
@@ -41,14 +43,18 @@ export const LeftPanel: FC<Partial<LeftPanelProps>> = memo(({ path }) => {
   const { openLeftPanel } = useExpandableFlyoutApi();
   const { eventId, indexName, scopeId, getFieldsData } = useLeftPanelContext();
   const eventKind = getField(getFieldsData('event.kind'));
+  const notesEnabled = useIsExperimentalFeatureEnabled('notesEnabled');
 
-  const tabsDisplayed = useMemo(
-    () =>
+  const tabsDisplayed = useMemo(() => {
+    const t =
```

Comment:
Could we rename `t` to sth more descriptive like `enabledTabs` or `tabList`?

Reply:
done!